echopype/calibrate/calibrate_ek.py

```diff
@@ -145,12 +145,16 @@ def __init__(self, echodata, env_params, cal_params, **kwargs):
         # Set the channels to calibrate
         # For EK60 this is all channels
         self.chan_sel = self.echodata[self.ed_group]["channel"]
+        beam = self.echodata[self.ed_group]
 
         # Get cal_params
         self.cal_params = get_cal_params_EK(
-            beam=echodata[self.ed_group].sel(channel=self.chan_sel),
-            vend=echodata["Vendor_specific"].sel(channel=self.chan_sel),
-            user_cal_dict=cal_params,
+            waveform_mode="CW",
+            freq_center=beam["frequency_nominal"],
+            beam=beam,
+            vend=self.echodata["Vendor_specific"],
+            user_dict=self.cal_params,
+            sonar_type="EK60",
```

Comment:
Probably good to set the `sonar_type` above as an attribute and use it here.

Reply:
Good idea. I added `sonar_type` as an attribute to all `Calibrate*` classes.